x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -37,9 +42,30 @@ export const EnginesList: React.FC = () => {
     });
   }, [meta.from, meta.size, throttledSearchQuery]);
 
+  if (fetchEngineApiStatus === Status.ERROR) {
```

Comment:
To handle error condition - reference from [engine_view](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_view.tsx#L39)

Reply:
I think this should be in the flyout and not in the page, as this error is specific to the flyout data.